src/Raven.Studio/typescript/components/pages/database/settings/documentSchema/partials/DocumentSchemaSelectActions.tsx

```diff
@@ -0,0 +1,242 @@
+import React from "react";
+import genUtils from "common/generalUtils";
+import { Icon } from "components/common/Icon";
+import { Checkbox } from "components/common/Checkbox";
+import { SelectionActions } from "components/common/SelectionActions";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import Button from "react-bootstrap/Button";
+import Dropdown from "react-bootstrap/Dropdown";
+import Spinner from "react-bootstrap/Spinner";
+import { useAppSelector } from "components/store";
+import { useDispatch } from "react-redux";
+import { useEventsCollector } from "hooks/useEventsCollector";
+import { documentSchemaSelectors } from "components/pages/database/settings/documentSchema/store/documentSchemaSliceSelectors";
+import {
+    documentSchemaActions,
+    DocumentSchemaValidatorConfig,
+} from "components/pages/database/settings/documentSchema/store/documentSchemaSlice";
+import useBoolean from "hooks/useBoolean";
+import DocumentSchemaDeleteModal from "components/pages/database/settings/documentSchema/partials/DocumentSchemaDeleteModal";
+import DocumentSchemaOperationConfirm, {
+    DocumentSchemaOperationConfirmType,
+} from "components/pages/database/settings/documentSchema/partials/DocumentSchemaOperationConfirm";
+import { useServices } from "hooks/useServices";
+import { databaseSelectors } from "components/common/shell/databaseSliceSelectors";
+import { documentSchemaUtils } from "components/pages/database/settings/documentSchema/documentSchemaUtils";
+import useConfirm from "components/common/ConfirmDialog";
+
+interface OperationConfirm {
+    type: DocumentSchemaOperationConfirmType;
+    onConfirm: () => void;
+    validators: DocumentSchemaValidatorConfig[];
+}
+
+export default function DocumentSchemaSelectActions() {
+    const dispatch = useDispatch();
+    const { reportEvent } = useEventsCollector();
+    const { value: isDeleteModalOpen, toggle: toggleDeleteModal } = useBoolean(false);
+    const { value: isTogglingStatus, setTrue: setTogglingStatus, setFalse: unsetTogglingStatus } = useBoolean(false);
+    const {
+        value: isTogglingGlobalStatus,
+        setTrue: setTogglingGlobalStatus,
+        setFalse: unsetTogglingGlobalStatus,
```

Comment:
This naming can be confusing. It suggest function with boolean argument. Use toggle if you can, or:

```suggestion
        setTrue: setIsTogglingGlobalStatusTrue,
        setFalse: setIsTogglingGlobalStatusFalse,
```